renovate.json5

```diff
@@ -0,0 +1,51 @@
+{
+  "extends": [
+    "config:recommended",
+    ":pinAllExceptPeerDependencies",
+  ],
+
+  "timezone": "Europe/London",
+
+  "dependencyDashboardApproval": false,
+  "prHourlyLimit": 0,
+  "prConcurrentLimit": 20,
+  "automerge": false,
+  "rangeStrategy": "pin",
+
+  "labels": ["dependencies"],
+
+  // opt-in to pre-commit beta feature
+  // https://docs.renovatebot.com/modules/manager/pre-commit/#enabling
+  "pre-commit": {
+    "enabled": true
+  },
+  "enabledManagers": [
+    "pep621",
+    "pre-commit"
+  ],
+
+  "packageRules": [
+    {
+      "matchManagers": ["pep621"],
+      "matchDepTypes": ["project.dependencies"],
+      "schedule": ["before 4am on monday"], // weekly
+    },
+    {
+      "matchManagers": ["pep621"],
+      "matchDepTypes": ["dependency-groups"],
+      "schedule": ["* 0-3 1 * *"], // monthly
+    },
+    {
+      "groupName": "ruff",
+      "matchManagers": ["pep621", "pre-commit"],
+      "matchPackageNames": ["ruff", "astral-sh/ruff-pre-commit"],
+      "schedule": ["* 0-3 1 * *"], // monthly
+    },
+    {
+      "groupName": "playwright",
+      "matchManagers": ["pep621"],
+      "matchPackageNames": ["playwright", "pytest-playwright"],
+      "schedule": ["* 0-3 1 * *"], // monthly
+    },
```

Comment:
Again, bump playwright and pytest-playwright together in the same PR. Same as we do on EE